controllers/webhook/runcompletion_event_trigger_unit_test.go

```diff
@@ -114,12 +117,27 @@ var _ = Context("EventDataToPbRunCompletion", func() {
 				Provider:              "some-provider",
 			}
 			Expect(protoRce).To(Equal(expectedResult))
+		})
+
+		It("returns empty slices when there are no artifacts", func() {
+			rceWithoutArtifacts := rce
+			rceWithoutArtifacts.Artifacts = []common.Artifact{}
+			rceWithoutArtifacts.ServingModelArtifacts = []common.Artifact{}
+
+			protoRce, err := RunCompletionEventToProto(rceWithoutArtifacts)
 			Expect(err).NotTo(HaveOccurred())
+
+			expectedArtifacts := []*pb.Artifact{}
```

Comment:
```suggestion
			emptySliceOfArtifacts := []*pb.Artifact{}
```

Reply:
done in f6752969d1d8488b7eaa89eb86bcc3c4b3a37521